cmd/root.go

```diff
@@ -58,6 +58,12 @@ func init() {
 		klog.Exitf("Failed to bind output-file flag: %v", err)
 	}
 
+	rootCmd.PersistentFlags().String("format", "json", "A format of the output which could be table or json. default json")
```

Comment:
I'm a bit confused here. If you specify `table` and an output file, does it make the file a csv by default? I think maybe the output file should output whatever the format is specified as, and we should have both `table` (or normal) and `csv`. Please correct me if I'm reading this wrong.

Reply:
You are reading this correctly. I think your comment makes sense, let me use that approach.